chore: update docs to reflect new package distribution (#2954)
telerik/blazor-docs: introduction.md

Page Templates offer ready-to-use Blazor layouts that effectively integrate the Telerik UI for Blazor components. These templates consist of Building Blocks, which are the individual elements that form the complete layouts. The templates and blocks streamline the development process by providing pre-designed, customizable elements. For more details, refer to the [Building Blocks and Page Templates documentation](https://www.telerik.com/design-system/docs/ui-templates/overview/).

## Trial and Commercial License

> UI for Blazor is distributed as a single package, with functionality determined by the applied license. By default, you can register for a free 30-day trial to explore the full capabilities of the library and receive technical support. During the trial period, a visual watermark and banner will appear over rendered components.

Reviewer: "single package" is technically incorrect, and "functionality determined by the applied license" may imply the trial version has limited features. Consider:

```suggestion
If you haven't purchased a UI for Blazor license yet, you can register for a free 30-day trial to explore the full capabilities of the library and receive technical support. During the trial period, a visual watermark and banner will appear over rendered components.
```

Author: Addressed in 2e51c21

> UI for Blazor is distributed as a single package, with functionality determined by the applied license. By default, you can register for a free 30-day trial to explore the full capabilities of the library and receive technical support. During the trial period, a visual watermark and banner will appear over rendered components.

Author: Addressed in 2e51c21
